x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_lifecycle/metadata.tsx

```diff
@@ -218,19 +212,28 @@ export function RetentionMetadata({
 function MetadataRow({
   metadata,
   value,
+  tip,
   button,
 }: {
   metadata: string;
   value: ReactNode;
-  action?: string;
+  tip?: string;
   button?: ReactNode;
 }) {
   return (
     <EuiFlexGroup alignItems="center" gutterSize="xl" responsive={false}>
       <EuiFlexItem grow={1}>
-        <EuiText>
-          <b>{metadata}</b>
-        </EuiText>
+        <EuiFlexGroup alignItems="center">
```

Comment:
nit: This has a gutterSize of m which feels like too much - can we reduce:
<img width="261" alt="Screenshot 2025-02-24 at 15 04 49" src="https://github.com/user-attachments/assets/d78bd414-4518-46d7-8181-51b0c3acc7ae" />

Reply:
Reduced to xs